pages/_components/partials/nav.njk

```diff
@@ -1,13 +1,20 @@
 <nav id="navbar" class="navbar">
+    <a class="d-none d-lg-block btn-emphasized btn btn-sm me-3" href="{{ 'news_href' | i18n }}">{{ 'news' | i18n }}</a>
     <ul>
         <li><a class="nav-link scrollto " href="#hero"><span>{{ 'section0' | i18n }}</span></a></li>
-                    <li><a class="nav-link scrollto " href="#tools"><span>{{ 'section1' | i18n }}</span></a></li>
-                    <li><a class="nav-link scrollto" href="#apps"><span>{{ 'section2' | i18n }}</span></a></li>
-                    <li><a class="nav-link scrollto" href="#about"><span>{{ 'section3' | i18n }}</span> </a></li>
-                    <li><a class="nav-link scrollto" href="#funByNumber"><span>{{ 'section4' | i18n }}</span></a></li>
-                    <li style="display:none;"><a class="nav-link scrollto" href="#usertest"><span>{{ 'section5' | i18n }}</span></a></li>
-                    <li><a class="nav-link scrollto" href="#contact"><span>{{ 'section6' | i18n }}</span></a></li>
-                    </ul>
+        <li><a class="nav-link scrollto " href="#tools"><span>{{ 'section1' | i18n }}</span></a></li>
+        <li><a class="nav-link scrollto" href="#apps"><span>{{ 'section2' | i18n }}</span></a></li>
+        <li><a class="nav-link scrollto" href="#about"><span>{{ 'section3' | i18n }}</span> </a></li>
+        <li><a class="nav-link scrollto" href="#funByNumber"><span>{{ 'section4' | i18n }}</span></a></li>
+        <li style="display:none;"><a class="nav-link scrollto" href="#usertest"><span>{{ 'section5' | i18n }}</span></a></li>
+        <li><a class="nav-link scrollto" href="#contact"><span>{{ 'section6' | i18n }}</span></a></li>
+        <li>
+            <a class="nav-link" href="{{ 'news_href' | i18n }}">
+                <i class="bi bi-box-arrow-up-right fs-4 me-3"></i>
+                {{ 'news' | i18n }}
+            </a>
+        </li>
```

Comment:
FMI, is there a reason to place the link to the news site after the "contact us" section? So as not to disrupt navigation between sections?

Reply:
Oh no, it does not affect navigation between sections at all but this is the only links which is an external link to it seems logic to myself that this link appears at the end of the list.

The only things required to navigate to an element is to fill the `href` attribute with the id of the targeted element.
e.g:

Reply:
Ah yes! I misspoke. I meant that putting a link in the middle of the sections could be confusing for the user navigating through the menu. Which you confirmed ;) Everything is clear to me now, thanks!
The PR seems fine but of course I let Samuel approve it.

Reply:
since it's external I would not include it here...

Reply:
But on mobile device, display the button next to the burger menu is not convenient.
Furthermore, as this is a subdomain, this is not exactly an external link (bad faith 😅)

Reply:
IC